policyengine_us/variables/gov/states/ar/tax/income/deductions/itemized/ar_itemized_deductions.py

```diff
@@ -0,0 +1,53 @@
+from policyengine_us.model_api import *
+
+
+class ar_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "AR itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=21"
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.irs.deductions
+        agi = tax_unit("adjusted_gross_income",period)
+        spouse_agi = tax_unit("spouse_separate_adjusted_gross_income",period)
+
```

Comment:
those are all part of itemized_deductions_less_salt - look at other PR's (itemized deductions)

Reply:
Thank you for your comment, Pavel. I read the itemized deductions in other PR and I'm still a bit confused about what deductions items are included in this itemized_deductions_less_salt. Does it include all the itemized deductions items like medical expense and interest expense so that we don't need to calculate the deduction for each item?